Makefile

```diff
@@ -181,6 +181,7 @@ UPROGS=\
 	_usertests\
 	_wc\
 	_zombie\
+	_test_project1\
```

Comment:
In line 184 **'_test_project1\'**  should be replaced **'_test_trace\'**  as the c file name is test_trace.c